fhir-server/src/main/java/au/csiro/pathling/update/ImportExecutor.java

```diff
@@ -137,6 +140,13 @@ public OperationOutcome execute(@Nonnull @ResourceParam final Parameters inParam
           .map(param -> ImportMode.fromCode(
               ((CodeType) param.getValue()).asStringValue()))
           .orElse(ImportMode.OVERWRITE);
+
+      // Get the serialized resource type from the source parameter.
+      final String format = sourceParam.getPart().stream()
```

Comment:
Sugar on top, but this could be put into an enum

Reply:
Fixed in 6a0b06b